lute/static/css/styles.css

```diff
@@ -987,21 +1041,19 @@ textarea {
 }
 
 /* focus on fields makes border same as tags */
-input:focus,
-textarea:focus,
-select:focus {
+form input:focus,
+form textarea:focus {
     border: 1px solid var(--tagify__tag-bg) !important;
     outline: none !important;
 }
 
-input[type="checkbox"]:focus,
-input[type="file"]:focus,
-button:focus {
```

Comment:
button:focus no longer needed?

Reply:
focus border on buttons didn't look that nice. removed
also there are some buttons we don't the border or outline for. easier to remove here than add exceptions